.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push]
+
+env:
+  CI: true
+
+jobs:
+  Test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install windows-build-tools
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: npm config set msvs_version 2019
+      - name: Install dependencies
+        run: npm i
+      - name: Run tests
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: npm test
```

In addition to some newer dependencies to build on Node 12+, I think the native modules need to be rebuild for Electron before running `electron-mocha`.

```suggestion
      - name: Install dependencies
        run: npm i
      - name: Rebuild dependencies for Electron
        run: npx electron-rebuild
      - name: Run tests
        uses: GabrielBB/xvfb-action@v1
        with:
          run: npm test
```

---

Works for me, on macOS and Ubuntu anyway. https://github.com/DeeDeeG/atom-keymap/runs/3003134465 Not sure what it takes to pass on Windows, but I notice the Travis CI didn't used to test on Windows, so this is new territory for this project's CI.